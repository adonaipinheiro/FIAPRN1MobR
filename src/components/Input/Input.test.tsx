import React from 'react';
import renderer from 'react-test-renderer';

import {Input} from './';

describe('<Input />', () => {
  it('Should take Snapshot', () => {
    const snapshot = renderer.create(<Input />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
