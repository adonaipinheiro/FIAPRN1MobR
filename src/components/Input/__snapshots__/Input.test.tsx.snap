// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Should take Snapshot 1`] = `
<View
  style={
    {
      "width": "100%",
    }
  }
>
  <TextInput
    autoCapitalize="none"
    autoComplete="off"
    autoCorrect={false}
    placeholderTextColor="#F2F2F290"
    style={
      {
        "backgroundColor": "#0002",
        "borderRadius": 4,
        "color": "#f2f2f2",
        "height": 40,
        "padding": 10,
      }
    }
  />
</View>
`;
